app/assets/javascripts/qsets.js

```diff
@@ -94,3 +94,11 @@ function initQuestionFilter() {
     $('.no-questions > p').html(noQuestionsNotification);
   }
 }
+
+function showNoSubsetNotification() {
+  if ($('.qset').length == 0) {
+    $('.no-qsets').removeClass("hidden");
+    var noSubsetsNotification = 'There are no subsets. ' + '<a href="#" data-toggle="modal" data-target="#modal-new-qset">Create one now!</a>';
```

Comment:
Right now it's not clear that "Create One Now" is a link until you hover over it. Not sure if we want to make it more obvious that it's clickable. I know that this is the design for when there are no questions too.